### PR TITLE
Adding back in the JES working disk

### PR DESCRIPTION
**Pull request by scottfrazer**
https://broadinstitute.atlassian.net/browse/DSDEEPB-1662

### CODE REVIEW

**Comment by geoffjentry** [comment]
Is this intended for review? Either way it's fine by me

**Comment by scottfrazer** [comment]
yes! please review!

File: src/main/scala/cromwell/logging/TerminalLayout.scala [hunk 1]
```diff
@@ -26,7 +29,7 @@ class TerminalLayout extends LayoutBase[ILoggingEvent] {
      */
     val prefix = if (event.getMessage == "{}") s"[${TerminalUtil.highlight(129, "*")}] " else ""
```
Comment (mcovarr) [inline src/main/scala/cromwell/logging/TerminalLayout.scala]: this is no longer used

Comment (scottfrazer) [inline src/main/scala/cromwell/logging/TerminalLayout.scala]: It isn't used but I also didn't want to lose this piece of knowledge (the fact that event.getMessage == `{}` for Akka actors).  Want me to comment it out or just remove it?  I guess it's really not that useful...

File: src/main/scala/cromwell/logging/TerminalLayout.scala [hunk 2]
```diff
@@ -17,6 +18,8 @@ class TerminalLayout extends LayoutBase[ILoggingEvent] {
       case x => x.toString.toLowerCase
     }
 
+    val timestamp = new SimpleDateFormat("YYYY/MM/dd HH:mm:ss:SS").format(new Date())
```
Comment (mcovarr) [inline src/main/scala/cromwell/logging/TerminalLayout.scala]: Any reason not to be consistent with the server date format?

`2015-10-20 12:02:49,318`

File: src/main/scala/cromwell/logging/TerminalLayout.scala (hunk 1 above)
Comment (mcovarr) [inline src/main/scala/cromwell/logging/TerminalLayout.scala]: I'd suggest getting rid of it, I had no idea until now that this was a thing in our logs.  :smile:

Comment (geoffjentry) [inline src/main/scala/cromwell/logging/TerminalLayout.scala]: Not specific to this but as a general rule I think commenting things out so they don't get forgotten needs an extremely strong argument to justify.

**Pull request merged**